deploy/olm-catalog/codeready-toolchain-operator/0.0.1/codeready-toolchain-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -58,7 +58,8 @@ spec:
     * Red Hat CodeReady Workspaces
     * OpenShift Pipelines
     ## Installation
-    CodeReady Toolchain Operator is a cluster-wide operator. It installs and configures the operators for the bundled developer tools to be available across all namespaces in the cluster. CodeReady Toolchain Operator and the developer tools get installed in a single namespace.
+    CodeReady Toolchain Operator is a cluster-wide operator. It installs and configures the operators for the bundled developer tools to be available across all namespaces in the cluster. CodeReady Toolchain Operator and OpenShift Pipelines get installed in single `openshift-operators` namespace and Red Hat CodeReady Workspaces in single `toolchain-che` namespace.
```

Comment:
Great! Thanks for taking care of this. @alexeykazakov is it what you request?

Reply:
@alexeykazakov same here as below: for now, we create a `toolchain-che` namespace for the Che-related OperatorGroup and Subscription resources.

Reply:
Yeah I know. But we need to fix it. IMO it's OK if we fix the namespace itself in the next PR.

Reply:
I've opened https://github.com/codeready-toolchain/toolchain-operator/pull/35 for that purpose